CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_amazon_ads_source v0.3.0
+[PR #17](https://github.com/fivetran/dbt_amazon_ads_source/pull/17) includes the following updates:
+## Feature update 🎉
+- Unioning capability! This adds the ability to union source data from multiple amazon_ads connectors. Refer to the [README](https://github.com/fivetran/dbt_amazon_ads_source/blob/main/README.md) for more details.
```

Comment:
Actually, can we remove the README "hide" sections in these updates. This way we are able to link directly to the section in the README where the users needs to understand how to configure the union feature.

Otherwise, I worry this cool feature is hidden from people actually using it.

Reply:
Updated.